mantle/kola/tests/ignition/luks.go

```diff
@@ -190,3 +246,8 @@ func luksSSST1Test(c cluster.TestCluster) {
 func luksSSST2Test(c cluster.TestCluster) {
 	runTest(c, true, 2, false)
 }
+
+// Verify that the rootfs is encrypted with Cex
+func luksCEXTest(c cluster.TestCluster) {
+	runCexTest(c, true)
+}
```

Comment:
Why have this and not directly point to `runCexTest`?

Reply:
Understood. I just followed the other luks test.

Reply:
Hi @jlebon ,

I am a bit stuck here.

 I've done the `CEX knob`  part and the replaced the test similar to the `runBasicTest` except using the `LocalTest` i  called the test function created `LUKSSanityCEXTest`.  How to call the `runCEXTest` here ? As  calling separate test `luks.cex`  causing complexity in pipeline. So here how can i achieve that?  
 
```
 register.RegisterTest(&register.Test{
		Run:             -> ?
		ClusterSize:   0,
		Name:          "basic.cex",   ->  ?
		Description:   "Verify that CEX-based rootfs encryption works.",
		Flags:         []register.Flag{},
		Distros:       []string{"rhcos"},
		Platforms:     []string{"qemu"},
		Architectures: []string{"s390x"},
		Tags:          []string{"luks", "cex", "reprovision"},
	})
```

Reply:
I'm not sure I understand the question. Your `Run` can be `RunCexTest`, right? You can the useless `cex` bool parameter to make it fit.

The current `Name` is fine (`luks.cex`).

Reply:
Oops, bool was creating the confusion. Done.